src/Illuminate/Validation/Rule.php

```diff
@@ -38,9 +38,9 @@ public static function exists($table, $column = 'NULL')
      * @param  bool  $array_keys
      * @return \Illuminate\Validation\Rules\In
      */
-    public static function in(array $values, bool $array_keys = false)
+    public static function in(array $values, bool $arrayKeys = null)
```

Comment:
I think you just remove the bool type hint since it's apparently supported only in php 7+ 😄 
http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration

Currently it's checking for a class bool:

> must be an instance of Illuminate\Validation\Rules\bool, boolean given

You can change them back to false after you remove the type hint...

Also you forgot to change the variable in the phpDoc, rest 👍